etc/kayobe/pulp.yml

```diff
@@ -164,6 +164,21 @@ stackhpc_pulp_distribution_deb_production:
 # Whether to sync CentOS Stream 8 packages.
 stackhpc_pulp_sync_centos_stream8: "{{ os_distribution == 'centos' }}"
 
+# Whether to sync Rocky Linux 8 packages.
+stackhpc_pulp_sync_rocky_8: "{{ os_distribution == 'rocky' }}"
+# Rocky 8 minor version number. Supported values: 6.
+stackhpc_pulp_sync_rocky_8_minor_version: 6
+# Rocky 8 Snapshot versions. The defaults use the appropriate version from
+# pulp-repo-versions.yml for the selected minor release.
+stackhpc_pulp_sync_rocky_8_appstream_version: "{{ lookup('vars', 'stackhpc_pulp_repo_rocky_8_%s_appstream_version' % stackhpc_pulp_sync_rocky_8_minor_version) }}"
+stackhpc_pulp_sync_rocky_8_baseos_version: "{{ lookup('vars', 'stackhpc_pulp_repo_rocky_8_%s_baseos_version' % stackhpc_pulp_sync_rocky_8_minor_version) }}"
+stackhpc_pulp_sync_rocky_8_extras_version: "{{ lookup('vars', 'stackhpc_pulp_repo_rocky_8_%s_extras_version' % stackhpc_pulp_sync_rocky_8_minor_version) }}"
+stackhpc_pulp_sync_rocky_8_nfv_version: "{{ lookup('vars', 'stackhpc_pulp_repo_rocky_8_%s_nfv_version' % stackhpc_pulp_sync_rocky_8_minor_version) }}"
+stackhpc_pulp_sync_rocky_8_powertools_version: "{{ lookup('vars', 'stackhpc_pulp_repo_rocky_8_%s_powertools_version' % stackhpc_pulp_sync_rocky_8_minor_version) }}"
```

Comment:
minor nit: I'd have gone with `s/sync/repo/` here - the sync variables are flags

Reply:
Gone with your suggestion here, thanks.